docker/gunicorn.conf.py

```diff
@@ -8,7 +8,7 @@
                if d.startswith("simplye_app-") and os.path.isdir(VENV_BASE_DIR / d)][0]
 
 # Shared Settings
-wsgi_app = "app:app"
+wsgi_app = "library_registry.app:app"
```

Comment:
@keithbauer was having issues with the Gunicorn process dying on start, because it wasn't finding the app object. Turns out I didn't update the path in `wsgi_app` to reflect making the app a package. Whoops!